main.py

```diff
@@ -13,45 +13,67 @@ class JournalEntry:
         title: str
         content: str
         date: datetime
-
-dict_journal_entry = {}
+        
+def add_entry(title: str, content: str) -> JournalEntry:
+    new_entry = JournalEntry(title=title,content=content,date=datetime.now())
+    #new_entry_dict = asdict(new_entry)
+    #new_entry = new_entry_dict
+    #print(new_entry)
+    return new_entry
+    #print(entries)
+    
+    #entries = load_entries()
+    #entries.append(new_entry_dict)
 
         
-def load_entries() -> str: 
-        with open('journal.json', "r") as user_file:
-         journal_content = user_file.read()
-        print(journal_content)
-"""Load journal entries from the JSON file."""
+        #Create a new journal entry.
 
-    
-def save_entries(entries: dict[JournalEntry]) -> str:
-        with open("journal.json","w") as json_file:
-             json.dump(dict_journal_entry,json_file)
-        """Save journal entries to the JSON file."""
+def save_entries(title: str, content: str) -> None: #entries: list[JournalEntry] #use asdict() in save_entries function
+        new_entry = add_entry(title,content)
+        with open(DB_FILE,"w") as f:
+             json.dump(asdict(new_entry),f,indent=2,default=str)
+        print("Serialized to JSON")
+        return
+"""Save journal entries to the JSON file."""
 
-    
-def add_entry(title, content) -> str:
-    save_entries(dict_journal_entry)
-    return
-    """Create a new journal entry and save it to the JSON file."""
 
+def load_entries() -> list[JournalEntry]: 
+        try:
+                with open(DB_FILE,"r") as f:
+                        data = json.load(f)
+                        print("Serizlied JSON data:", data)
+        except FileNotFoundError:
+                        data = []
+        finally:
+               print("Your journal entries were successfully stored and loaded") #Using finally just to show my exception handling is working"""
+"""Load journal entries from the JSON file."""
     
-def list_entries(entries) -> str:
-    #print(dict_journal_entry)
-    return
-    """Print all journal entries to the console."""
+def list_entries(entries: list[JournalEntry]) -> None:
+        entries = load_entries()
+        print("My second test of deserialized entries: ", entries)
+        if not entries:
+          print("No entries found")
```

Comment:
here the indent has one space, I think Python will throw an error here, because it relies on a consistent number of indents